syscalls.go

```diff
@@ -262,3 +263,15 @@ var haveBPFToBPFCalls = internal.NewFeatureTest("bpf2bpf calls", "4.16", func()
 	_ = fd.Close()
 	return nil
 })
+
+var haveSyscallWrapper = internal.NewFeatureTest("syscall wrapper", "4.17", func() error {
+	testSyscallName, ok := internal.PlatformPrefix("sys_bpf")
+	if !ok {
+		return internal.ErrNotSupported
```

Comment:
I think it would be more appropriate to return an error that is not ErrNotSupported here, since we're not actually able to tell whether the arch has as wrapper.